Tmedia 838 image arrows counter

JackHowa opened the pull request:
## Ticket

- [TMEDIA-838](https://arcpublishing.atlassian.net/browse/TMEDIA-838) sub-task of https://arcpublishing.atlassian.net/jira/software/c/projects/TMEDIA/boards/875?modal=detail&selectedIssue=TMEDIA-689&assignee=5e387d6a1b1d910e5dfd7a9b

## Description

Additional image arrows that can be toggled to show and hide via theme file 

## Acceptance Criteria

Image arrows near the image counter drop off at the mobile breakpoint (the large arrows on the image itself stay)

## Test Steps

1. Checkout branch - `git checkout TMEDIA-838-image-arrows-counter`
2. Run Storybook `npm run storybook`
3. See the additional default buttons story. Make sure you're on commerce theme
![Screen Shot 2022-04-19 at 13 43 54](https://user-images.githubusercontent.com/5950956/164073819-55e1dc71-1c37-404d-b841-87ffe7678272.png)
4. Click next ">" arrow and see the other previous "<" show. This can be spaced by the theme file 
![Screen Shot 2022-04-19 at 13 44 55](https://user-images.githubusercontent.com/5950956/164073916-8f4c99b8-477f-4b7f-a2a9-4efb40bf1532.png)
5. Go to the custom additional controls story. 
6. See the various updates to the inline custom buttons
![Screen Shot 2022-04-19 at 13 46 41](https://user-images.githubusercontent.com/5950956/164074234-bc83b688-4c4f-4e76-b246-9846b4cec13e.png)

![Screen Shot 2022-04-19 at 13 46 54](https://user-images.githubusercontent.com/5950956/164074251-c5b24812-f02e-4c7f-b9b2-0cdf8cf27aca.png)
7. In the commerce theme, shrink the screen to mobile. See the additional arrows no longer visible via display: none

## Author Checklist

_The author of the PR should fill out the following sections to ensure this PR is ready for review._

- [x] Confirmed all the test steps a reviewer will follow above are working.
- [x] Ran this code locally and checked that there are not any unintended side effects. For example, that a CSS selector is scoped only to a particular block.
- [x] Confirmed relevant documentation has been updated/added.
- [x] Add label - **ready for review** when the pull request is ready for someone to begin reviewing

## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._

- [ ] All GitHub Actions pass
- [ ] Ran the code locally based on the test instructions.
- [ ] Checked Chromatic for Storybook changes, accepted the updates if acceptable
- [ ] Looked to see that the new or changed code has code coverage, specifically. We want the global code coverage to keep on going up with targeted testing.
- [ ] Approve and Add label - **ready to merge** if you are happy with the pull request
- [ ] Want another reviewer? Add the label **additional review**

JackHowa committed:
feat: Add additional carousel controls buttons which may be hidden on mobile view

JackHowa committed:
docs: Show and hide additional on desktop and mobile

JackHowa committed:
docs: Fix name not matching

matthewroach commented:
@JackHowa - Same question/comment as on the other PR - https://github.com/WPMedia/arc-themes-components/pull/73#issuecomment-1103794266

matthewroach commented:
@JackHowa - Same question/comment as on the other PR - https://github.com/WPMedia/arc-themes-components/pull/73#issuecomment-1103794266

JackHowa committed:
Merge branch 'arc-themes-release-version-2.00' into TMEDIA-838-image-arrows-counter

JackHowa committed:
fix: Ensure correct order of elements per feedback

Per https://github.com/WPMedia/arc-themes-components/pull/74#issuecomment-1103795243

JackHowa commented:
@matthewroach updated

> @JackHowa - Same question/comment as on the other PR - [#73 (comment)](https://github.com/WPMedia/arc-themes-components/pull/73#issuecomment-1103794266)

<img width="1436" alt="Screen Shot 2022-04-20 at 09 17 19" src="https://user-images.githubusercontent.com/5950956/164251305-7931eee6-a62b-4d09-b602-40652b03b208.png">

nschubach reviewed (approved):
Looks great to me, covers the requirement.

JackHowa committed:
feat: Add default for aligning items next to each other and centering

JackHowa commented:
@matthewroach updated to align items within the controls container next to each other horizontally rather than vertically

matthewroach commented on src/components/carousel/index.jsx:
```diff
@@ -29,6 +29,28 @@ const DefaultPreviousButton = ({ id, onClick }) => (
+		label="Previous Slide"
+		className={`${COMPONENT_CLASS_NAME}__button ${COMPONENT_CLASS_NAME}__button--additional-previous`}
+	>
+		{"<"}
```
Can use use an `<Icon />` here? `ChevronLeft`

JackHowa replied:
sure. importing one component into the other ok?

matthewroach replied:
Good question - we've not doing it yet - maybe we use an HTML entity then rather than the text >

JackHowa replied:
@matthewroach updated
![Screen Shot 2022-04-20 at 16 20 04](https://user-images.githubusercontent.com/5950956/164324908-caf6e35c-be9e-4fe8-8f9a-5feb89d5785e.png)

JackHowa replied:
@matthewroach updated 
![Screen Shot 2022-04-20 at 16 20 04](https://user-images.githubusercontent.com/5950956/164326857-2f719802-0c5c-4364-8414-c161d87d7c34.png)

matthewroach commented on src/components/carousel/index.jsx:
```diff
@@ -29,6 +29,28 @@ const DefaultPreviousButton = ({ id, onClick }) => (
+		label="Next Slide"
+		className={`${COMPONENT_CLASS_NAME}__button ${COMPONENT_CLASS_NAME}__button--additional-next`}
+	>
+		{">"}
```
Can use use an `<Icon />` here? `ChevronLeft`

JackHowa committed:
feat: Use icon component for default left and right on top actions area

Per comment https://github.com/WPMedia/arc-themes-components/pull/74#discussion_r854559950

JackHowa commented:
fixing test for ><

JackHowa committed:
test: Update tests based on icon change
